### PR TITLE
feat: scaffolding blocks package

### PR DESCRIPTION
**Pull request by guilean**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** 
https://impulsumvc.atlassian.net/browse/COM-589

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

**Storybook Link** (Link to a local storybook url to review new functionality)

### CODE REVIEW

File: packages/blocks/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage",
+    "type:check": "yarn tsc --project tsconfig.json --noEmit",
+    "type:emit": "tsc --project ./tsconfig.json -d --declarationDir lib --declarationMap --emitDeclarationOnly && find lib -type f \\( -name '*.test.*' -o -name '*.stories.*' \\) -exec rm {} +"
+  },
+  "peerDependencies": {
+    "react": "^18.1.0"
+  },
+  "dependencies": {
+    "@cmpsr/components": "1.20.3"
```
Comment (tkanzakic) [inline packages/blocks/package.json:32]: should this be a peer dependency?

File: packages/blocks/src/config/jest/assetTransformer.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+
+export const assetTransformer = {
```
Comment (tkanzakic) [inline packages/blocks/src/config/jest/assetTransformer.ts:3]: why do you need this?

File: packages/blocks/src/tests/renderHookWithProviders.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import { ComposerProvider } from '@cmpsr/components';
+
+export function renderHookWithProviders<TProps, TResult>(callback: (props: TProps) => TResult) {
```
Comment (tkanzakic) [inline packages/blocks/src/tests/renderHookWithProviders.tsx:5]: do you need this?

File: packages/blocks/package.json (hunk 1 above)
Comment (guilean) [reply packages/blocks/package.json:30]: Moved to peer dependency

File: packages/blocks/src/config/jest/assetTransformer.ts (hunk 2 above)
Comment (guilean) [reply packages/blocks/src/config/jest/assetTransformer.ts]: Unneeded, removed

File: packages/blocks/src/tests/renderHookWithProviders.tsx (hunk 3 above)
Comment (guilean) [reply packages/blocks/src/tests/renderHookWithProviders.tsx]: Unneeded, removed file + library

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**